### PR TITLE
Adding pyasdf receipe

### PR DESCRIPTION
**Pull request by krischer**
Adds a receipe for http://seismicdata.github.io/pyasdf/index.html

### CODE REVIEW

File: recipes/pyasdf/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [py36]
```
Comment (jakirkham) [inline recipes/pyasdf/meta.yaml]: Why this skip?

**Comment by jakirkham** [comment]
One question, though a non-blocking one, so am merging.

**Comment by jakirkham** [comment]
Thanks @krischer.

**Comment by krischer** [comment]
> Why this skip?

A dependency does not yet have Python 3.6 packages. I'll update it in the feedstock once those are available.


> Thanks @krischer.

This is my package so I really need to be the one thanking you :)

**Pull request merged**